.azure-pipelines/code-oob.yml

```diff
@@ -0,0 +1,157 @@
+parameters:
+- name: TargetBranch
+  displayName: Branch this module will generated to
+  type: string
+  default: main
+- name: ServiceName
+  displayName: Service to OOB release
+  type: string
+  default: Databricks
+- name: Preview
+  displayName: Is preview version
+  type: boolean
+  default: false
+stages:
+- stage: OOBRelease
+  displayName: OOBRelease
+  jobs:
+  - job: OOBRelease
+    timeoutInMinutes: 180
+    pool: pool-windows-2019
+    steps:
+    - task: DotNetCoreCLI@2
+      displayName: Build
+      inputs:
+        command: custom
+        custom: msbuild
+        arguments: 'build.proj /t:Clean;Build /p:Configuration=Release'
+    - task: UseDotNet@2
+      inputs:
+        version: '3.1.x'

```

Comment:
The .NET is for what purpose? Currently we have upgraded to .NET 6. .NET core 3.1 is only for TestFx